kf_lib_data_ingest/etl/configuration/target_api_config.py

```diff
@@ -5,6 +5,8 @@
 which contains, in the order that you want them to be loaded into the target
```

Comment:
I'll just put this comment up here since it applies to the loader v1 and v2 docstrings. Its hard to know which methods of the class are required vs not. The only one I know is not required is the transform_records_list bc the docstring says so

Reply:
Hmm. Proposed fix?

Reply:
How about adding something at the top of each method's docstring? `OPTIONAL` or `REQUIRED`. I know we talked about this but I forget the explanation. Why not use abstract classes to mandate what should be implemented?

Reply:
> Why not use abstract classes to mandate what should be implemented?

As far as I know, abc methods don't enforce call signatures.